modules/trusted-firmware-m/nordic_nrf/include/device_cfg.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef DEVICE_CFG_H__
+#define DEVICE_CFG_H__
+
+#include <autoconf.h>
+
+/* ARRAY_SIZE causes a conflict as it is defined both by TF-M and indirectly by devicetree.h */
+#undef ARRAY_SIZE
+#include <zephyr/devicetree.h>
+
+#if DOMAIN_NS == 1U
+#define TFM_UART uart0
+#endif
+
+#if DOMAIN_NS != 1U
+#define TFM_UART uart1
+#endif
+
+#define DEFAULT_UART_BAUDRATE DT_PROP_OR(DT_NODELABEL(TFM_UART), current_speed, 115200)
+
+#if DT_PROP_OR(DT_NODELABEL(TFM_UART), hw_flow_control, 0) == 1
```

Comment:
nit: `DT_NODE_HAS_PROP()` always returns 1 for boolean properties, so this will anyway call `DT_PROP()`
```suggestion
#if DT_PROP(DT_NODELABEL(TFM_UART), hw_flow_control)
```

Reply:
Not sure I understood the argument, DT_NODE_HAS_PROP is not used here, but I can see that they behave the same so I will take your suggestion.

Reply:
`DT_PROP_OR()` calls `DT_NODE_HAS_PROP()` internally, and if that returns 1, `DT_PROP()` is called.